cps-global-lib/README.md

```diff
@@ -13,18 +13,24 @@ Every workflow script in your Jenkins see these shared library scripts in their
 The directory structure of the shared library repository is as follows:
 
     (root)
-     +- src                 # groovy source files
-         +- org
-             +- foo
-                 +- Bar.groovy  # for org.foo.Bar class
-
+     +- src                     # groovy source files
+     |   +- org
+     |       +- foo
+     |           +- Bar.groovy  # for org.foo.Bar class
+     +- vars
+         +- foo.groovy          # for global 'foo' variable/function 
+         +- foo.txt             # help for 'foo' variable/function
```

Comment:
Not documented sufficiently, should mention transform of configured markup formatter.